cli/commands/migrate_apply.go

```diff
@@ -57,7 +57,7 @@ func newMigrateApplyCmd(ec *cli.ExecutionContext) *cobra.Command {
   hasura migrate apply --goto 125
   
   # Apply all down migrations upto version 125, last applied is 150
-  hasura migrate apply --goto 125
+  hasura migrate apply --type down --goto 125
```

Comment:
This seems to be documented as expected, the `--type` flag is not expected to be used in conjunction with `--goto`